.travis.yml

```diff
@@ -20,7 +20,9 @@ jdk:
   - oraclejdk8
 
 before_install:
-- echo "sdk.dir=$ANDROID_HOME" > local.properties
+  - echo "sdk.dir=$ANDROID_HOME" > local.properties
+  - mkdir "$ANDROID_HOME/licenses" || true
+  - cp ./android-sdk-licenses/* "$ANDROID_HOME/licenses/"
```

Comment:
We might not need this change because of bintray-release plugin

Reply:
👀